src/decode.rs

```diff
@@ -3465,12 +3457,6 @@ fn decode_sb(
     let mut by8 = 0;
     let ctx = match pass {
         FrameThreadPassState::First(ts_c) => {
-            if false && bl == BlockLevel::Bl64x64 {
-                println!(
-                    "poc={},y={},x={},bl={:?},r={}",
-                    frame_hdr.frame_offset, t.b.y, t.b.x, bl, ts_c.msac.rng,
-                );
-            }
```

Comment:
Please don't delete this.  It's currently dead code, but just because it's been disabled.  It shouldn't be deleted.